docker/bookkeeper/entrypoint.sh

```diff
@@ -1,38 +1,46 @@
 #!/bin/bash
 #
-# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-set -e
 
-# To create directories for multiple ledgers and journals if specified
-create_dir() {
-  IFS=',' read -ra directories <<< $1
-  for i in "${directories[@]}"
-  do
-      mkdir -p $i
-      if [ "$(id -u)" = '0' ]; then
-          chown -R "${BK_USER}:${BK_USER}" $i
-      fi
-  done
-}
+#
+# Note: This file contains source code copied from the Apache BookKeeper
+#       project (http://bookkeeper.apache.org). Specifically, it contains
+#       source code copied from file `init_bookie.sh`, revision `3bfecd4`:
+#  github.com/apache/bookkeeper/blob/branch-4.7/docker/scripts/init_bookie.sh.
+#
+
+# Exit the script in case of an error
+set -e
 
 BOOKIE_PORT=${bookiePort:-${BOOKIE_PORT}}
 BOOKIE_PORT=${BOOKIE_PORT:-3181}
+BOOKIE_HTTP_PORT=${BOOKIE_HTTP_PORT:-9090}
```

Comment:
Should we turn this off using BK_httpServerEnabled config?

Reply:
I've modified the default port to the one used in Bookkeeper image.